### PR TITLE
Add FAQ to gh enterprise page

### PR DESCRIPTION
**Pull request by twinkarma**

### CODE REVIEW

File: pages/training/gh-enterprise.md [hunk 1]
```diff
@@ -26,8 +26,9 @@ not a member of UoS should be marked as an external collaborator.
 
 #### 1.1.2 Ownership of Repositories
 
-All repositories under UoS Github Enterprise must be owned by a member of UoS, with the permission of the owner or their
-delegates of the Github Organisation in which it belongs to.
+All repositories under UoS Github Enterprise must be owned by a member of UoS, with the permission of the owner (or their
+delegates) of the Github Organisation in which it belongs to. Additional owners of the repository can an external
```
Comment (ns-rse) [inline pages/training/gh-enterprise.md]:
```suggestion
delegates) of the Github Organisation in which it belongs to. Additional owners of the repository can be external
```

File: pages/training/gh-enterprise.md [hunk 2]
```diff
@@ -26,8 +26,9 @@ not a member of UoS should be marked as an external collaborator.
...
+collaborator to UoS.
```
Comment (ns-rse) [inline pages/training/gh-enterprise.md]:
```suggestion
collaborators to UoS.
```

**Review by ns-rse — CHANGES_REQUESTED** [review]
Minor grammar suggestions, otherwise good to merge.

**Comment by twinkarma** [comment]
Thanks @ns-rse !

**Review by ns-rse — APPROVED** [review]
Good to go :+1:

**Pull request merged**